modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vmi-yaml-ssh_{context}"]
+= Accessing a virtual machine instance via SSH with YAML configurations
+
+You can enable a SSH connection to a VMI without the need to run the `virtctl expose` command. The YAML file for the VMI should include a label with a value that is identical to the value for the `selector` in the service YAML file. Then, the service forwards the SSH traffic connections to the VMI.
+
+The following examples show the configurations for the VMI's YAML file and the service YAML file.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+* You must be in the same project as the VMI.
+* The VMI you want to access must be connected to the default pod network by using the `masquerade` binding method.
+
+.Procedure
+. In the YAML file for the VMI, add the label and a value for exposing the service for SSH connections. Add the port number and enable the `masquerade` feature for the interface.
+
++
+.Example output
+[source,yaml]
+----
+...
+kind: VirtualMachineInstance
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-ssh
+  name: vmi-ssh <1>
```

Comment:
I think that the <1> should point to the label `special: vmi-ssh` and not the name of the vmi (`vmi-ssh`). Maybe in addition an explanation about the name could be added as well, but the name of the vmi is not that important. I think that the label is the thing that might need clarification. 

also the attached explanation (line 60) should not include explanation on the vmi name, but on the label. So I think you can leave only this part: 

> This label is used by the service to identify the virtual machine instances that are enabled for SSH traffic connections.

maybe add the explanation about the vmi-name separately.  

Also maybe worth mentioning that this specific label is not required, and it could be any key: value pair, that will appear under 'labels' in the vmi yaml, and under 'selector' in the service yaml.

Reply:
Edited this content. Please review.

Reply:
Reworded this content. You make a good point about the label. Please review.